demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -53,7 +53,7 @@ class ContributionConverter:
 
     def __init__(self, name: str = '', contribution: Union[str] = None, description: str = '', author: str = '',
                  gh_user: str = '', create_new: bool = True, pack_dir_name: Union[str] = None,
-                 base_dir: Union[str] = None, no_pipenv: bool = False):
+                 base_dir: Union[str] = None, no_pipenv: bool = False, readme_files: List[str] = []):
```

Comment:
not sure i follow why you add it as class argument and not an attribute, will we ever initalize it?